### PR TITLE
Feature/addsArcheota

### PR DESCRIPTION
**Pull request by cordhernandez**
BlackSourceLabs/BlackNectar-iOS#43

### CODE REVIEW

File: BlackNectar/BlackNectar/AppDelegate.swift [hunk 1]
```diff
@@ -20,13 +21,28 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
         AromaClient.TOKEN_ID = "34576d0b-6060-4666-9ac1-f5a09be219c3"
         
         AromaClient.beginMessage(withTitle: "App Launched")
             .addBody("Build #\(AppDelegate.buildNumber)")
             .withPriority(.low)
             .send()
         
+        NSSetUncaughtExceptionHandler() { error in
+            
+            AromaClient.beginMessage(withTitle: "App Crashed")
```
Comment (SirWellington) [inline BlackNectar/BlackNectar/AppDelegate.swift:36]: Nice!

File: BlackNectar/BlackNectar/GlobalLocationManager.swift [hunk 2]
```diff
@@ -46,18 +46,24 @@ class UserLocation: NSObject, CLLocationManagerDelegate, MKMapViewDelegate  {
...
          self.onLocation = callback
         locationManager.startUpdatingLocation()
+        
     }
     
     func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) {
         
         guard let location: CLLocation = locations.first else {
+            
+            LOG.error("Failed to Update First Location")
```
Comment (SirWellington) [inline BlackNectar/BlackNectar/GlobalLocationManager.swift:59]: Maybe include the value in `locations`

**Review by SirWellington — COMMENTED** [review]
Smooth as silk

File: BlackNectar/BlackNectar/GlobalLocationManager.swift (hunk 2 above)
Comment (cordhernandez) [reply BlackNectar/BlackNectar/GlobalLocationManager.swift:59]: ok got it !

**Pull request merged**